docs/source/user_guide/calculation_plugins/pw.rst

```diff
@@ -154,12 +159,13 @@ Quantum ESPRESSO pw.x plugin (note that most of them apply also to the
 cp.x plugin).
 
 While the input link with name 'parameters' is used for the content of the 
-Quantum Espresso namelists, additional parameters can be specified in the 'settings' input, also as Dict.
+Quantum Espresso namelists, additional parameters can be specified in the 'settings' input, also as 
+:py:class:`Dict <aiida.orm.nodes.data.dict.Dict>`.
 
 After having defined the content of ``settings_dict``, you can use
```

Comment:
you can use it as input of a calculation ``calc`` by doing::
change to
you can use it as an input by adding it to the process builder::